src/tests/java/picard/sam/testers/SamFileTester.java

```diff
@@ -340,4 +342,8 @@ private File createInputFile() {
     public SamReader getInput() {
         return samRecordSetBuilder.getSamReader();
     }
+
+    public void setRecordsNeedSorting(final boolean recordsNeedSorting) {
```

Comment:
could this return the instance of `SamFileTester` to allow chaining?  Then the name could be `recordsNeedSorting(...)`.